pkg/storage/status/status.go

```diff
@@ -0,0 +1,12 @@
+// Package status declares error constants returned by the variou
+// implementations of the Store interface.
+package status
```

Comment:
Why not have it in the storage package?

Reply:
@kerneltime because it creates a new constraint re not having cyclical dependencies. With a separate case package i am not adding any such implicit constraint on thid pkg